hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/migration/BackfillFailedEthereumTransactionContractResultMigrationTest.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.mirror.importer.migration;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.hedera.mirror.common.domain.contract.ContractResult;
+import com.hedera.mirror.common.domain.transaction.EthereumTransaction;
+import com.hedera.mirror.common.domain.transaction.Transaction;
+import com.hedera.mirror.importer.EnabledIfV1;
+import com.hedera.mirror.importer.IntegrationTest;
+import com.hedera.mirror.importer.repository.ContractResultRepository;
+import com.hederahashgraph.api.proto.java.ResponseCodeEnum;
+import java.io.File;
+import lombok.RequiredArgsConstructor;
+import lombok.SneakyThrows;
+import org.apache.commons.io.FileUtils;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.test.context.TestPropertySource;
+
+@EnabledIfV1
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+@Tag("migration")
+@TestPropertySource(properties = "spring.flyway.target=1.79.0")
+class BackfillFailedEthereumTransactionContractResultMigrationTest extends IntegrationTest {
+
+    private static final byte[] EMPTY = new byte[] {};
+
+    private final ContractResultRepository contractResultRepository;
+
+    private final JdbcTemplate jdbcTemplate;
+
+    @Value("classpath:db/migration/v1/V1.79.1__backfill_ethereum_transaction_contract_result.sql")
+    private final File sql;
```

Comment:
Prefer Spring's `Resource` and treat it as an InputStream.
```suggestion
    private final Resource sql;
```